.travis.yml

```diff
@@ -0,0 +1,24 @@
+---
+
+language: node_js
+
+node_js:
+  - 10/*
+
+sudo: required
+
+services:
+  - docker
```

Comment:
Docker isn't used in this app

Reply:
Docker is used to run the elementary project's tool on Travis CI.